server/src/main/java/io/crate/expression/symbol/Symbols.java

```diff
@@ -49,7 +49,7 @@
     public static final Predicate<Symbol> IS_GENERATED_COLUMN = input -> input instanceof GeneratedReference;
 
     public static List<DataType> typeView(List<? extends Symbol> symbols) {
-        return Lists.transform(symbols, Symbol::valueType);
+        return Lists2.map(symbols, Symbol::valueType);
```

Comment:
Maybe also a caze for `lazyMap`, or otherwise we should also change the function name, because the `View` implied that the structure is lazy.